tests/ui/perfherder/alerts_test.jsx

```diff
@@ -76,14 +76,21 @@ const mockModifyAlert = {
 const mockUpdateAlertSummary = (alertSummaryId, params) => ({
   failureStatus: null,
 });
-const alertsView = () =>
+const alertsView = (updateParams = () => {}, id) =>
   render(
     <AlertsView
+      validated={{
+        hideDwnToInv: undefined,
+        hideImprovements: undefined,
+        hideAssignedToOthers: undefined,
+        filter: undefined,
+        updateParams,
+      }}
       user={testUser}
       projects={repos}
       location={{
         pathname: '/alerts',
-        search: '',
+        search: id ? 'id=20174' : '',
```

Comment:
I think `id=20174` it's a little bit too magical.
Why this?

Reply:
I forgot it there while doing some tests. I'll remove it